backend/cmd/headlamp.go

```diff
@@ -1186,9 +1333,11 @@ func (c *HeadlampConfig) deleteCluster(w http.ResponseWriter, r *http.Request) {
 
 func (c *HeadlampConfig) addClusterSetupRoute(r *mux.Router) {
 	// We do not support this feature when in-cluster
-	if c.useInCluster {
+	if !c.enableDynamicClusters {
+		fmt.Println("Dynamic clusters endpoints: disabled.")
 		return
 	}
+	fmt.Println("Dynamic clusters endpoints: enabled.")
```

Comment:
Is there any specific reason for adding the enableDynamicClusters flag? The existing logic already makes sure that the dynamic clusters doesn't work for incluster mode.

Reply:
This was answered in today's call.

Reply:
Answered in yesterday's call. Resolving the conversation.